pkg/proxy/iptables/proxier.go

```diff
@@ -47,7 +47,7 @@ import (
 	"k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/proxy"
 	"k8s.io/kubernetes/pkg/proxy/healthcheck"
-	utilproxy "k8s.io/kubernetes/pkg/proxy/util"
+	proxyutil "k8s.io/kubernetes/pkg/proxy/util"
```

Comment:
Why change this?

Reply:
Other imports like "k8s.io/apiserver/pkg/AAA/BBB", the short name always is AAABBB, so I think we may follow this order better.
So, according to them the short name of "k8s.io/apiserver/pkg/util/feature", should be `proxyutil`.

Reply:
Hmmm, pkg/proxy/iptables/proxier.go has the same lines,

see: utilproxy "k8s.io/kubernetes/pkg/proxy/util"